<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZhSoft</class>
 <widget class="QMainWindow" name="ZhSoft">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ZhSoft</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="tableViewMsaDisplay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>256</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tableViewCrossing">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>321</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMoveUp">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMoveDown">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>240</y>
      <width>321</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Croisement des données</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelReperage">
         <property name="text">
          <string>Rubrique &quot;réparage :&quot;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxCrossing"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="labelCrossing">
         <property name="text">
          <string> Nom de la table à croiser :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditCrossing">
         <property name="text">
          <string>ZhPasFix_Croisement</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Croiser deux données</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>767</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionOuvrir"/>
    <addaction name="actionActualiser"/>
    <addaction name="actionQuiter"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOuvrir"/>
   <addaction name="separator"/>
   <addaction name="actionActualiser"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOuvrir">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/rec/icon/new.jpg</normaloff>:/rec/icon/new.jpg</iconset>
   </property>
   <property name="text">
    <string>Ouvrir</string>
   </property>
  </action>
  <action name="actionActualiser">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/rec/icon/reload.jpg</normaloff>:/rec/icon/reload.jpg</iconset>
   </property>
   <property name="text">
    <string>Actualiser</string>
   </property>
  </action>
  <action name="actionQuiter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ressource.qrc"/>
 </resources>
 <connections/>
</ui>
